### PR TITLE
Polkadart scale codec documentation

### PR DESCRIPTION
**Pull request by justkawal**
## Description
* Adds documentation for polkadart_scale_codec

Documentation Preview can be seen at: https://github.com/rankanizer/polkadart/blob/justkawal/polkadart-scale-codec-documentation/packages/polkadart_scale_codec/README.md

### CODE REVIEW

File: packages/polkadart_scale_codec/README.md [hunk 1]
```diff
@@ -1 +1,344 @@
-## polkadart_scale_codec
+# polkadart_scale_codec
+
+### [polkadart_scale_codec](https://www.pub.dev/packages/polkadart_scale_codec) is a flutter and dart library for encoding and decoding types supported by **polkadot**.
```
Comment (leonardocustodio) [inline packages/polkadart_scale_codec/README.md:3]: Let's not make this text giant and bold

File: packages/polkadart_scale_codec/README.md [hunk 2]
```diff
@@ -1 +1,344 @@
...
+```
+
+### 2. Install it
+
+You can install packages from the command line:
+
+with `pub`:
+
+```css
+  dart pub get
```
Comment (leonardocustodio) [inline packages/polkadart_scale_codec/README.md:23]: There are some extra spaces

File: packages/polkadart_scale_codec/README.md [hunk 3]
```diff
@@ -1 +1,344 @@
...
+```
+
+with `Flutter`:
+
+```css
+  flutter pub get
```
Comment (leonardocustodio) [inline packages/polkadart_scale_codec/README.md:29]: Extra spaces

File: packages/polkadart_scale_codec/README.md [hunk 4]
```diff
@@ -1 +1,344 @@
...
+```
+
+### 3. Import it
+
+Now in your `Dart` code, you can use:
+
+```dart
+    import 'package:polkadart_scale_codec/polkadart_scale_codec.dart';
```
Comment (leonardocustodio) [inline packages/polkadart_scale_codec/README.md:37]: Extra spaces

**Review by leonardocustodio — CHANGES_REQUESTED** [review]

File: packages/polkadart_scale_codec/README.md (hunk 4 above)
Comment (justkawal) [reply packages/polkadart_scale_codec/README.md:37]: Done ✅

File: packages/polkadart_scale_codec/README.md (hunk 2 above)
Comment (justkawal) [reply packages/polkadart_scale_codec/README.md:23]: Done ✅

File: packages/polkadart_scale_codec/README.md (hunk 1 above)
Comment (justkawal) [reply packages/polkadart_scale_codec/README.md:3]: Done ✅

Comment (justkawal) [reply packages/polkadart_scale_codec/README.md:3]: Only left the title giant and bold.

File: packages/polkadart_scale_codec/README.md [hunk 5]
```diff
@@ -1 +1,345 @@
...
+--- | ---
+Unsigned Int | `u8, u16, u32, u64, u128, u256`
+Signed Int | `i8, i16, i32, i64, i128, i256`
+String | `Text`
+Boolean | `bool`
+Address | `Address`
+Bytes | `Bytes`
+Compact | `Compact<T>`
+Enum | `_enum`
+Struct | `_struct`
```
Comment (Lohann) [inline packages/polkadart_scale_codec/README.md:51]: Hmm, why underscore before enum and struct?

**Review by Lohann — CHANGES_REQUESTED** [review]
I don't see an example for `struct`

File: packages/polkadart_scale_codec/README.md (hunk 5 above)
Comment (justkawal) [reply packages/polkadart_scale_codec/README.md:51]: `_enum` was used by every chain's schema.

**Comment by justkawal** [comment]
> I don't see an example for `struct`

Adding example for struct.

**Review by pastre — APPROVED** [review]

**Review by gabrielokura — APPROVED** [review]

**Pull request merged**